### PR TITLE
Changed TORQUE to PBS

### PR DESCRIPTION
**Pull request by Charlottez112**
## Description
Changed TORQUE to PBS because the existing Torque scheduler works decently well with other PBS (qsub) systems that are not using Torque.

## Motivation and Context
See Issue #388 

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [x] I have updated the API documentation as part of the package doc-strings.
- [x] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

File: flow/environment.py
Comment (bdice) [inline flow/environment.py:37]: For consistency with `LSFScheduler`, I would name this `PBSScheduler`.

**Comment by bdice** [comment]
@Charlottez112 Thanks for the PR! I pushed a couple commits that finish up this transition. There are two remaining steps:
- [ ] Add yourself to `contributors.yaml`
- [ ] Review this complementary PR to signac-docs: https://github.com/glotzerlab/signac-docs/pull/127

**Comment by Charlottez112** [comment]
> * [ ]  Review this complementary PR to signac-docs: [glotzerlab/signac-docs#127](https://github.com/glotzerlab/signac-docs/pull/127)

Thank you for finishing this! I didn't realize there're files in signac that we need to change

**Review by bdice — APPROVED** [review]

**Comment by bdice** [comment]
@b-butler @Tobias-Dwyer This can be merged after a second approval.

**Review by b-butler — APPROVED** [review]
LGTM

**Pull request merged**
